src/Command.js

```diff
@@ -18,7 +18,11 @@ export default class Command {
     this.repository = new Repository();
     this.progressBar = progressBar;
     this.concurrency = (!flags || flags.concurrency === undefined) ? DEFAULT_CONCURRENCY : Math.max(1, +flags.concurrency || DEFAULT_CONCURRENCY);
-    this.toposort = !flags || (flags.sort == null ? true : flags.sort);
+
+    const {sort} = this.getOptions();
+
+    // If the option isn't present then the default is to sort.
+    this.toposort = sort == null || sort;
```

Comment:
@evocateur Good thought!  The constructor name is actually pulled from the object bound to `this` rather than the class where the method is defined, so it should work as intended in this case.

Here's a small script to demonstrate:

```js
class Foo {
  constructor() {
    console.log(this.constructor.name);
  }
}

class Bar extends Foo {}

new Bar;
```

This prints `Bar` rather than `Foo`, since that's what is instantiated.

Thanks for thinking through this so carefully!